src/storage/table.c

```diff
@@ -1667,6 +1667,16 @@ void flecs_table_delete(
                 }
 
                 ecs_move_t move_dtor = ti->hooks.move_dtor;
+                
+                // if move_ctor & move is not set and ctor_move_dtor & move_dtor is set,
+                // use move_dtor as ctor_move_dtor. The reason we
```

Comment:
Shouldn't this say "use ctor_move_dtor as move_dtor"?

Reply:
should it? since we are using the "move_dtor", but it's using underneath "ctor_move_dtor" , hence as? 

I might be wrong, it's definitely a word twist

Reply:
committed a change to the comment due to the word twist.